index.html

```diff
@@ -0,0 +1,177 @@
+                        <h5>FAQs</h5>
+                        <p>If you have gone throuh the programs requirement and have random questions, you can explore our FAQs for more clarity.</p>
+                    </div><br>
+                </div>
+                <button class="register-btn" type="button">Click here to register</button>
+                <a class="prog" href="#">SEE THE PROGRAM</a>
+            </section>
+            <section class="section3">
+                 <p>Guest Speakers and host</p>
+                 <hr class="newhr">
+            <div class="speaker-container">
+<!-- 
+                 <div class="speakers">
+                     <img class="images" src="images/Benimg.jpeg" alt="speakers imge">
+                     <div class="column">
+                     <h5>Ben Ghana</h5>
+                     <h4>Founder of the SSAMB </h4>
+                     <hr>
+                     <p>Ben Ghana founder of the SSAMB community. Putting all efforts to see girls are well oriented about tech. Tech is a master piece for ladies a well valued craft</p>
+                     </div>
+                 </div>
+
+                 <div class="speakers">
+                     <img class="images" src="images/Benimg.jpeg" alt="speakers imge">
+                     <div class="column">
+                     <h5>Mike Nigeria</h5>
+                     <h4>Co-founder of the SSAMB </h4>
+                     <hr>
+                     <p>In my opinion, women are tech are a big asset to the selves and the community. So I will encourage every girl to get into the tech industry and explore options.</p>
+                     </div>
+                 </div>
+
+                 <div class="speakers">
+                     <img class="images" src="images/Benimg.jpeg" alt="speakers imge">
+                     <div class="column">
+                     <h5>Steph Bahamas</h5>
+                     <h4>Co-founder of the SSAMB </h4>
+                     <hr>
+                     <p>Tech is the leading industry in the world. It comes with magically powers and well cherished when women invade the tech space.</p>
+                     </div>
+                 </div>
+
+                 <div class="speakers">
+                     <img class="images" src="images/Benimg.jpeg" alt="speakers imge">
+                     <div class="column">
+                     <h5>Steph Bahamas</h5>
+                     <h4>Co-founder of the SSAMB </h4>
+                     <hr>
+                     <p>The sole idea of advocating for black girls in tech is to bring value to every black girl and let them see the potential in them.</p>
+                     </div>
+                 </div>
+
+                 <div class="speakers">
+                    <img class="images" src="images/Benimg.jpeg" alt="speakers imge">
+                    <div class="column">
+                    <h5>Jane Cameroon</h5>
+                    <h4>Guest</h4>
+                    <hr>
```

Comment:
Please do not leave unused code as a comment in your codebase. Unused code as comments makes your code not clean.
Kindly remove all commented code in your index.html file 😊